### PR TITLE
[AJ-522] add an argument for integration test update

### PR DESCRIPTION
**Pull request by jordansafer**
**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Workbench utility libraries, built for 2.13. You can find the full list of packa
 
 Contains common classes and utilities for writing tests against Workbench REST web services.
 
-Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-service-test" % "2.0-5863cbd" % "test" classifier "tests"`
+Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-service-test" % "2.0-TRAVIS-REPLACE-ME" % "test" classifier "tests"`
```
Comment (calypsomatic) [inline README.md:119]: `TRAVIS-REPLACE-ME` is a version?

Comment (jordansafer) [reply README.md:119]: My understanding is there's a github action that compiles the code, generates a new version, and then substitutes it into the README

Comment (jordansafer) [reply README.md:119]: All of this happens after the code is merged to develop-

Comment (jordansafer) [reply README.md:119]: See here: https://github.com/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md#versioning

Comment (calypsomatic) [reply README.md:119]: Weird.  Ok

**Review by calypsomatic — APPROVED** [review]

**Review by davidangb — APPROVED** [review]

**Pull request merged**